tests/unit/test_graph.py

```diff
@@ -19,14 +19,16 @@
 from dbt.contracts.graph.manifest import MacroManifest, ManifestStateCheck
 from dbt.graph import NodeSelector, parse_difference
 from dbt.events.functions import setup_event_logger
+from dbt.flags import set_from_args
+from argparse import Namespace
```

Comment:
nit: this should really be back up where it was with the built-in imports in the first section of the imports. Likewise, `from .utils ...` should be moved down here.

Reply:
Removed the addition of set_from_args. It can be fixed with the ticket to do that to all of the tests.